docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -25,86 +25,63 @@ image::images/integrations-server-hosted-container.png[Hosted {integrations-serv
+. In {kib}, go to **Management > {fleet} > Settings**. For more information
+about these settings, see {fleet-guide}/fleet-settings.html[{fleet} settings].
 // lint ignore fleet-server
-. Under *Fleet Server hosts*, click *Edit hosts* and specify one or more host
+. Under **Fleet Server hosts**, click **Edit hosts** and specify one or more host
 URLs your {agent}s will use to connect to {fleet-server}. For example,
 `https://192.0.2.1:8220`, where `192.0.2.1` is the host IP where you will
 install {fleet-server}. Save and apply your settings.
 
-. In the *{es} hosts* field, specify the {es} URLs where {agent}s will send data.
+. In the **{es} hosts** field, specify the {es} URLs where {agent}s will send data.
 For example, `https://192.0.2.0:9200`. Skip this step if you've started the
 {stack} with security enabled (you cannot change this setting because it's
 managed outside of {fleet}).
 
 . Save and apply the settings.
 
-. Click the *Agents* tab and follow the in-product instructions to add a
+. Click the **Agents** tab and follow the in-product instructions to add a
 {fleet} server:
 +
 [role="screenshot"]
 image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
 
-*Notes:*
+**Notes:**
 
-* Make sure you download an x64 architecture installation package.
-
-* For the agent policy, choose one with a {fleet-server} integration, or click
-*Create policy* to create one now. Alternatively you can
+* Choose **Quick Start** if you want {fleet} to generate a
+{fleet-server} policy and enrollment token for you.
+* Choose **Advanced** if you want to either:
+** Use your own {fleet-server} policy. You can create a new {fleet-server}
+policy or select an existing one. Alternatively you can
 {fleet-guide}/create-a-policy-no-ui.html[create a {fleet-server} policy without using the UI],
-then select the policy here.
-
-* If you choose *Production* deployment mode, learn how to generate certs in
-{fleet-guide}/secure-connections.html[Configure SSL/TLS for self-managed
-{fleet-server}s].
-
+and select the policy here.
+** Use your own TLS certificates to encrypt traffic between {agent}s and
+{fleet-server}. To learn how to generate certs, refer to
+{fleet-guide}/secure-connections.html[Configure SSL/TLS for self-managed {fleet-server}s].
 * It's recommended you generate a unique service token for each
 {fleet-server}. For other ways to generate service tokens, see
 {ref}/service-tokens-command.html[`elasticsearch-service-tokens`].
-
+* If you are providing your own certificates:
+** Before running the `install` command, make sure you replace the values in
+angle brackets. 
+** Note that the URL specified by `--url` must match the DNS name used to
+generate the certificate specified by `--fleet-server-cert`.
 * The `install` command installs the {agent} as a managed service and enrolls it
-in a {fleet-server} policy. For example, the following command installs
-a {fleet-server} and uses self-signed certs:
-+
-[source,yaml]
```

Comment:
I've removed the examples here because it duplicates what the user sees in the UI but without the contextual info that the UI provides. I think these examples might just confuse users rather than clarify.